src/Core/ScreenshotCache.vala

```diff
@@ -1,141 +1,152 @@
-public class AppCenterCore.ScreenshotCache {
+/*-
+ * Copyright 2020 elementary, Inc. (https://elementary.io)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+public class AppCenterCore.ScreenshotCache : GLib.Object {
     private const int MAX_CACHE_SIZE = 100000000;
 
     public string screenshot_path;
```

Comment:
Since it looks like we assign this in the construct block, should this be:
```suggestion
    public string screenshot_path { get; private set; }
```